packages/angular-ivy/package.json

```diff
@@ -50,12 +50,8 @@
     "build:syncSymlinks": "ts-node ./scripts/syncSourceFiles.ts",
     "circularDepCheck": "madge --circular src/index.ts",
     "clean": "rimraf build coverage sentry-angular-ivy-*.tgz",
-    "fix": "run-s fix:eslint fix:prettier",
-    "fix:eslint": "eslint . --format stylish --fix",
-    "fix:prettier": "prettier --write \"{src,test,scripts}/**/**.ts\"",
-    "lint": "run-s lint:prettier lint:eslint",
-    "lint:eslint": "eslint . --format stylish",
-    "lint:prettier": "prettier --check \"{src,test,scripts}/**/**.ts\"",
+    "fix": "eslint . --format stylish --fix",
+    "lint": "eslint . --format stylish",
```

Comment:
q: So I understand we replace prettier with biome. Why are we then removing the prettier scripts from the individual packages and not replacing them with biome scripts?

Reply:
Because running biome in all codebase is faster than running it individually